### ISSUE TITLE
Add support for right-clicking

### ISSUE THREAD

**Issue by qguv** [BlueM/cliclick#31]
The pure X11 (Linux) equivalent to cliclick, called xdotool, allows the user to specify which mouse button is used when sending a click event.

**Comment by BlueM** [comment]
Right-clicking is possible by using the traditional “click with Ctrl key pressed” approach:
`cliclick kd:ctrl c:. ku:ctrl`

However, this might not be too obvious. (And there might be peculiar target applications where right-clicking is *not* the same as Ctrl-clicking.) I’ll see if it can be implemented directly. Probably it’s not a big deal.

**Comment by muellermartin** [comment]
I've added a new handler in PR #42 - maybe you guys want to check this out ;)
